WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportFormMetadataProvider.swift

```diff
@@ -39,6 +40,12 @@ class SupportFormMetadataProvider {
     /// Common system & site  tags. Used in Zendesk Forms.
     ///
     func systemTags() -> [String] {
+        if stores.isAuthenticated == false,
+           AuthenticatorAnalyticsTracker.shared.state.lastFlow == .loginWithSiteAddress,
+           AuthenticatorAnalyticsTracker.shared.state.lastStep == .usernamePassword {
```

Comment:
Do you think we can inject the singleton like we do for all other dependencies? 😇

Reply:
IMO we injected those singletons for testing purposes. I looked into `AuthenticatorAnalyticsTracker` and it seems it's not possible to mock it with either subclassing or extensions since `state` is tied to the type. Unless I'm missing something, injecting the tracker doesn't bring any benefit.

I cleaned up this code a bit by creating a local variable and avoiding the duplication of calling the shared instance in 32387642c4d4a07b4f5f46e0481934ad814a7a66.